### ISSUE TITLE
what does the event interface received bandwidth usage over net device speed max mean

### ISSUE THREAD

**Issue by alsoeric** [netdata/netdata#5805]
##### Question summary
I'm getting warnings and critical notices of a few network parameters. I am not able to find them in Google. Clarification/definition would be appreciated.

##### OS / Environment
rhel
##### Expected results
I'm getting a series of errors and warnings from netdata with the following message:

>🔴 chopin is critical - 1m received traffic overflow
> net.p8p1 (p8p1) 
>1m received traffic overflow = 91% 
>interface received bandwidth usage over net device speed max

This message comes from health.d/net .conf template 1m_received_traffic_overflow. Have been googling and not finding anything except links to the netdata alarm itself. what does the event indicate, what causes this event and how do I fix it?

**Comment by ilyam8** [comment]
Hi @alsoeric 

This alarm

https://github.com/netdata/netdata/blob/d9d41e6b9e23db9aa48481dd7efa4c317273c9fb/health/health.d/net.conf#L17-L30


It warns you that average usage (rx direction) for the last minute of your network interface (p8p1) is 91% (warn threshold is 85)

**Comment by Ferroin** [comment]
Building a bit on what @ilyam8 said (which is correct), the alarm itself is comparing raw network traffic to what the kernel thinks the network interface's speed is (which it's usually correct about for wired or fiber Ethernet links, but may be wildly wrong for wireless links).  This ratio, often referred to as 'link saturation', is a rough indication of how close your system is to being performance limited by network bandwidth (100% means you can't transfer data any faster than you already are over that link).

Realistically, this alarm means that your system will be less able to handle bursts of inbound network traffic on the specified interfaces.  Whether or not this is actually an issue is kind of dependent on what you're using the system for.  If, for example, the server is responsible for managing or storing backups from other systems, seeing this alarm fire whenever backups are running is not necessarily an indication that something is wrong, it just means your backup performance is limited by network bandwidth.  OTOH, if you see this on a back-end webserver that's behind a load-balancing reverse proxy, something is probably wrong (either the load-balancer isn't working right, or one stream is hitting you with a huge amount of data).

**Comment by alsoeric** [comment]
this makes a lot of sense. I watched  the alarm channel (my telegram group) when backups ran and I got those alerts.  I'm going to limit the borg data rate and free up 25% of capacity.  I'll see if that resolves the too many alerts problem.  Since I'm running multiple borg backups at the same time, I suspect I'll still fill the pipe at the NAS and internet (local and remote backup targets). At least the interhost connections will have capacity for the "real work" and I can debug the dropped packets problem.
